.github/stale.yml

```diff
@@ -0,0 +1,52 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: false
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: false
+
+# Set to true to ignore issues with an assignee (defaults to false)
+exemptAssignees: false
+
+# Label to use when marking as stale
+staleLabel: stale
+
+# Comment to post when removing the stale label.
+# unmarkComment: >
+
+closeComment: >
+This issue/pull request has been automatically closed due to inactivity and no response to the auto close bot
+request. If you feel this is inappropriate, please respond to this issue requesting it to be re-opened. 
+
+# Limit the number of actions (marking or closing issues/PRs) per hour, from 1-30. Default is 30
+limitPerRun: 30
+
+# Issue specific handling:
+issues:
+  daysUntilStale: 60
+  markComment: >
+    This issue has been automatically marked as stale because it has not had
+    recent activity. It will be closed if no further activity occurs. Thank you
+    for your contributions.
+   exemptLabels:
+     - limitation
+     - security
+     - P1
```

Comment:
I wonder if we need `P2` here too?

Reply:
We don’t have an active policy around the labels yet and if it’s a priority and hasn’t been touched in 2 months, I question the label. Wdyt?

Reply:
wfm - let's leave as just P1 for now and get this landed.